src/components/Page.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+import Counter from './Counter';
+import Buttons from './Buttons';
+
+export default function Page() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
+
+  function handleClick({ number }) {
+    setState({
+      count: count + number,
+    });
+  }
+
+  return (
+    <div>
+      <p>Hello World!</p>
+      <Counter
+        count={count}
+        onClick={() => handleClick({ number: 1 })}
```

Comment:
```suggestion
        onClick={handleClick}
```

실제로 함수 호출은 `Counter` 컴포넌트 내부에서 일어나는게 관심사의 분리 관점에서 더 좋을거에요.
지금 `Page` 컴포넌트에서는 `Counter` 컴포넌트의 `onClick` 이벤트가 어떻게 실행되는지 알필요가 없기 때문이에요.
여기서는 단순히 함수를 전달해 주는 역할만하는거죠.